routes/facetsWithArticles.js

```diff
@@ -13,6 +13,70 @@ router.get('/test', async (req, res, next) => {
 	res.render("facetsWithArticles/test", { facetsJson: JSON.stringify(results) } );
 });
 
+router.get('/clusteredImages/one', async (req, res, next) => {
```

Comment:
it feels like these routes could be regrouped using  a variable path `/clusteredImages/:template` where template triggers the right rendering.

Reply:
And possibly a query param or path var for people/topics/orgs